aws/resource_aws_cloudtrail.go

```diff
@@ -72,6 +73,46 @@ func resourceAwsCloudTrail() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validateArn,
 			},
+			"event_selector": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 5,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"read_write_type": &schema.Schema{
+							Type:         schema.TypeString,
+							Required:     true,
```

Comment:
This attribute should be:
```go
Optional: true,
Default: cloudtrail.ReadWriteTypeAll,
```

To match the [CloudTrail API documentation](https://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_EventSelector.html)